### PR TITLE
[ELASTIC-177] Fix Kibana test failure on 1.9.

### PR DESCRIPTION
**Pull request by mpereira**

### CODE REVIEW

File: frameworks/elastic/tests/config.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def wait_for_expected_nodes_to_exist(service_name=SERVICE_NAME, task_count=DEFAU
     stop_max_delay=DEFAULT_ELASTIC_TIMEOUT*1000,
     retry_on_result=lambda res: not res)
 def check_kibana_plugin_installed(plugin_name, service_name=SERVICE_NAME):
-    cmd = "bash -c '$MESOS_SANDBOX/kibana-$ELASTIC_VERSION-linux-x86_64/bin/kibana-plugin list'"
+    cmd = "KIBANA_DIRECTORY=$(ls -d $MESOS_SANDBOX/kibana-*-linux-x86_64); $KIBANA_DIRECTORY/bin/kibana-plugin list"
```
Comment (elezar) [inline frameworks/elastic/tests/config.py]: The `;` is not strictly required, right?

File: testing/sdk_cmd.py [hunk 2]
```diff
@@ -271,12 +271,19 @@ def task_exec(task_name: str, cmd: str, return_stderr_in_stdout: bool = False) -
...
+        # so we inject these manually.
+        environment_variables = [
+            "LIBPROCESS_IP=0.0.0.0",
+            "MESOS_SANDBOX={}".format(sandbox_path)
+        ]
+
+        if cmd.startswith("./"):
+            cmd = os.path.join(sandbox_path, cmd)
+
+        full_cmd = "bash -c '{}; {}'".format(' '.join(environment_variables), cmd)
```
Comment (elezar) [inline testing/sdk_cmd.py]: Is there a semantic difference when including the `;`?

Comment (mpereira) [reply testing/sdk_cmd.py]: Yes. Prepending env var assignments to a command doesn't reflect on the actual command. Bash seems to expand `$A` before the assignment happens. Breaking it into two commands makes it work. 💩 

```
$ dcos task exec kibana bash -c 'A=1 echo $A'

$ dcos task exec kibana bash -c 'A=1; echo $A'
1
```

File: frameworks/elastic/tests/config.py (hunk 1 above)
Comment (mpereira) [reply frameworks/elastic/tests/config.py]: It is in this case, same issue as in the comment below.

File: testing/sdk_cmd.py (hunk 2 above)
Comment (mpereira) [reply testing/sdk_cmd.py]: > Have you run this PR against a 1.9 cluster?
I have.

**Comment by mpereira** [comment]
@elezar: Have you run this PR against a 1.9 cluster?

I have.

**Pull request merged**